tests/conftest.py

```diff
@@ -497,90 +497,81 @@ def fixture_core_components(fixture_config):
+        "leaf_turnover_c_p_ratio": [415.0, 327.4, 554.5, 380.9],
+        "plant_reproductive_tissue_turnover_c_p_ratio": [125.5, 105.0, 145.0, 189.2],
+        "root_turnover_c_p_ratio": [656.7, 450.6, 437.3, 371.9],
+        "litter_consumption_above_metabolic": [0.019785, 0.011631, 0.016129, 0.023456],
+        "litter_consumption_above_structural": [0.02097, 0.01609, 0.01019, 0.00988],
+        "litter_consumption_woody": [0.4773833, 0.385701, 0.373456, 0.162192],
+        "litter_consumption_below_metabolic": [0.010373, 0.005794, 0.010181, 0.013494],
+        "litter_consumption_below_structural": [0.013547, 0.011674, 0.012738, 0.009168],
+        "herbivory_waste_leaf_carbon": [3e-5, 2.1e-3, 2.85e-3, 2.7e-3],
+        "herbivory_waste_leaf_nitrogen": [23.1, 33.5, 23.1, 17.3],
+        "herbivory_waste_leaf_phosphorus": [212.5, 344.8, 334.8, 420.1],
+        "herbivory_waste_leaf_lignin": [0.13, 0.08, 0.27, 0.22],
     }
 
-    for var_name, var_values in data_values.items():
-        data[var_name] = DataArray(var_values, dims=["cell_id"])
-
-    # The layer dependent data has to be handled separately - at present all of these
-    # are defined only for the topsoil layer
-    lyr_str = fixture_core_components.layer_structure
-
-    data["soil_moisture"] = lyr_str.from_template()
-    data["soil_moisture"][lyr_str.index_all_soil] = np.array(
-        [
-            [232.61550125, 196.88733175, 126.065797, 75.63195175],
-            [66.248474, 194.91137, 121.29988, 52.04422],
-        ]
-    )
+    for var, vals in pool_values.items():
+        data[var] = DataArray(vals, dims=["cell_id"])
 
-    data["matric_potential"] = lyr_str.from_template()
-    data["matric_potential"][lyr_str.index_all_soil] = np.array(
-        [[-3.0, -10.0, -250.0, -10000.0], [-2.8625, -8.978, -137.8, -8553.25]]
-    )
+    # Vertically structured variables
+    data["soil_temperature"] = lyr_strct.from_template()
+    data["soil_temperature"][lyr_strct.index_topsoil] = 20
+    data["soil_temperature"][lyr_strct.index_subsoil] = [19.5, 18.7, 18.7, 17.6]
 
-    data["soil_temperature"] = lyr_str.from_template()
-    data["soil_temperature"][lyr_str.index_all_soil] = np.array(
-        [[35.0, 37.5, 40.0, 25.0], [22.5, 22.5, 22.5, 22.5]]
-    )
+    # At present the soil model only uses the top soil layer, so this is the
+    # only one with real test values in
+    data["matric_potential"] = lyr_strct.from_template()
+    data["matric_potential"][lyr_strct.index_topsoil] = [-10.0, -25.0, -100.0, -100.0]
+    data["matric_potential"][lyr_strct.index_subsoil] = [-11.0, -29.5, -123.0, -154.1]
 
-    data["air_temperature"] = lyr_str.from_template()
-    data["air_temperature"][lyr_str.index_filled_atmosphere] = np.array(
+    data["air_temperature"] = lyr_strct.from_template()
+    data["air_temperature"][lyr_strct.index_filled_atmosphere] = np.array(
         [30.0, 29.844995, 28.87117, 27.206405, 16.145945]
     )[:, None]
```

Comment:
If the 16.14 C affects your litter activities negatively feel free to change that, this value is too low and an artifact from an old implementation

Reply:
This stuff purely gets used for unit testing, so no problem with that!